src/sage/rings/number_field/number_field.py

```diff
@@ -4126,18 +4126,78 @@ def completely_split_primes(self, B=200):
             sage: K.<xi> = NumberField(x^3 - 3*x + 1)
             sage: K.completely_split_primes(100)
             [17, 19, 37, 53, 71, 73, 89]
+
+        TESTS:
+
+        This checks that the example at :issue:`28113` is fixed::
+
+            sage: K.<a> = QuadraticField(17)
+            sage: K.completely_split_primes(20)
+            [2, 13, 19]
+
+        This checks that the example at :issue:`32982` is fixed::
+
+            sage: x = polygen(QQ)
+            sage: f = x^3 - x^2 + 2*x + 8
+            sage: K.<a> = NumberField(f)
+            sage: K.completely_split_primes(80)
+            [2, 59, 73, 79]
+
+        This checks a case where the index of Z[a] in OK is greater than 1::
+
+            sage: K.<a> = NumberField(x^2-13)
+            sage: K.completely_split_primes(80)
+            [3, 17, 23, 29, 43, 53, 61, 79]
+
+        This checks that extra primes in the polynomial discriminant are not a factor::
+
+            sage: K.<a> = NumberField(x^2-13*9*49)
+            sage: K.completely_split_primes(20)
+            [3, 17]
+
+        This checks that we're still OK in the case of fields of non-prime degree::
```

Comment:
```suggestion
        This checks that the method is correct in the case of fields of non-prime degree::
```
OK conflicts with $O_K$ here